sms_plusserver.py

```diff
@@ -175,7 +182,12 @@ def configure(self, **kwargs):
 
     # High-level API:
 
-    def send(self, sms, timeout=None, fail_silently=False):
+    def send(
+        self,
+        sms: 'SMS',
+        timeout: Optional[float] = None,
```

Comment:
Shouldn't it be of an `int` type? Regarding docs: `:param timeout: network timeout in seconds` it sounds like a solid `int` :)
(here and in other `timeouts` instances)

Reply:
ha... I also thought that on first place, but `sleep` can take values like: `0.5` and it is `float`. 

Secondary I go really deeper into `requests` docs (`requests/sessions.py:493`) and got this:
```python
class Session:

    def request():

        :param timeout: (optional) How long to wait for the server to send
            data before giving up, as a float, or a :ref:`(connect timeout,
            read timeout) <timeouts>` tuple.
```
that's why I used `float`